### DISCUSSION TITLE
Has SeleniumJupiter.class of wdm been tested with Selenide

### DISCUSSION THREAD

**Discussion by pratyushdeshpande** [selenide/selenide#1734] [General]
Hello,

I stumbled across SeleniumJupiter.class of wdm which manages the browsers for each test with minimal/reduced boilerplate code. I believe Selenide uses wdm , but has it been tested to work well with SeleniumJupiter.class.

**Comment by asolntsev** [comment]
@pratyushdeshpande Yes, Selenide uses WDM. No, I personally didn't try to use Selenide with SeleniumJupiter. Are you getting any problems?
